src/libbsdl/include/BSDL/config.h

```diff
@@ -0,0 +1,123 @@
+// Copyright Contributors to the Open Shading Language project.
+// SPDX-License-Identifier: BSD-3-Clause
+// https://github.com/AcademySoftwareFoundation/OpenShadingLanguage
+
+
+#pragma once
+
+#ifndef BSDL_NS
+#    define BSDL_NS bsdl
+#endif
+
+#define BSDL_ENTER_NAMESPACE namespace BSDL_NS {
+#define BSDL_LEAVE_NAMESPACE }
+
+#ifndef BSDL_INLINE_METHOD
+#    define BSDL_INLINE_METHOD inline
+#endif
+#ifndef BSDL_INLINE
+#    define BSDL_INLINE inline
+#endif
```

Comment:
Just curious why?

Reply:
This just gives the BSDL consumer an opportunity to override the inlining and __device__ and other things. See testrender overrides in bsdl_config.h